src/main/java/game/jdbc/picturepoker/GameDAO.java

```diff
@@ -45,6 +40,23 @@ public Game findById(long id){
         return game;
     }
 
+    public ArrayList<Long> getPIDsByGame(Game dto){
```

Comment:
Thank you for making a getPID function
I will probably update all of my PID get functions accordingly.